### ISSUE TITLE
COE - Cleanup Old Objects App - Cannot select users.

### ISSUE THREAD

**Issue by otecomputer** [microsoft/coe-starter-kit#5492]
When I load the "Cleanup Old Objects app" in the COE toolkit I am unable to view anyone when I try to use the "Select User" option.

I see myself on the right however the person on the left is not me in the list and I cannot expand on the list.

![CleaupOldObjectsScreenShot](https://github.com/microsoft/coe-starter-kit/assets/57675454/56a11f2e-cf01-4dfe-afdd-63ed52186370)

**Comment by Jenefer-Monroe** [comment]
Expected behavior here is that on load you see the logged in users information, and the list of their empoloyees

![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b95b0063-65e5-438c-b7fd-b55cda06a08f)

When you click on an employee you see their information
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/7d729d8a-316b-4314-bf5f-3794e5441adb)

**Comment by otecomputer** [comment]
Ah, so the adjustment if possible would be if the administrator of the power platform could see everyone. I am managing the whole thing at the moment and in the current configuration the tool is not useful then.

**Comment by Jenefer-Monroe** [comment]
Yes agreed. This app was a stop gap really while we waited for the MDA / custom pages world to be ready (which it is now). 
So I'll add this as a requirement for that feature. 
Thanks for posting!

**Comment by mohamrizwa** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.
New Issue Reference: https://github.com/microsoft/coe-starter-kit/issues/10331
Thank you for your contributions
